pages/dashboard.tsx

```diff
@@ -31,23 +27,42 @@ export default function DashboardPage() {
     }
   }, [ready, authenticated, router]);
 
-  async function delegateandauth() {
+  async function delegateAndSend() {
     if (!client) {
       console.error("No client yet");
       return;
     }
+
     setIsLoading(true);
     try {
-      const uoHash = await client.sendUserOperation({
-        uo: {
-          target: "0xc0ffee254729296a45a3885639AC7E10F9d54979",
-          value: 0n,
-          data: "0x0",
+      const result = await client.sendCalls({
```

Comment:
you can just do this if you want
```suggestion
      const { preparedCallIds: [callId] } = await client.sendCalls({
```